.vale/styles/Localization/PhrasalVerbs.yml

```diff
@@ -0,0 +1,25 @@
+---
+extends: substitution
+ignorecase: true
+level: warning
+link: https://redhat-documentation.github.io/vale-at-red-hat/docs/main/reference-guide/termswarnings/
+message: "Avoid using phrasal verbs. Consider using '%s' rather than '%s'."
+scope: sentence
+action:
+  name: replace
+swap:
+  # Start each error message with "Do not use ..."
+  # Error messages must be single quoted.
```

Comment:
I don't get this comment.

It seems related to the error message,not the swap section

Reply:
@aireilly Could you please help clarify this rule?